proto/alliance/events.proto

```diff
@@ -1,5 +1,5 @@
 syntax = "proto3";
-package alliance.alliance;
+package alliance;
```

Comment:
@ThanhNhann why'd the name of the proto package change here?

Reply:
(and in the other .proto files)

Reply:
I see that Alliance's struct proto folder doesn't have submodule folder so I think it's correct to change it like now

Reply:
Nice.  Can you confirm @javiersuweijie ?

Reply:
Seems correct but I'll need to check if this is a breaking change that updates the GRPC types for the alliance models.